src/WpfMath/SourceSpan.cs

```diff
@@ -21,16 +22,22 @@ public SourceSpan(string source, int start, int length)
 
         public string Source { get; }
 
+        /// <summary>A name identifying the source (e.g. a user input, or some sort of macro body).</summary>
+        public string SourceName { get; }
```

Comment:
It doesn't clear what names are allowed here. Is it finite set of names and could be described as enum instead, or something more generic? Please clarify.

Reply:
My intention was to make any names allowed: user may pass them directly to the parser. They have purely informative function.